### DISCUSSION TITLE
Straw too high to bale

### DISCUSSION THREAD

**Discussion by Krayell** [Courseplay/Courseplay_FS22#1861] [Q&A] [answered]
When harvesting straw crops, the harvester leaves a huge straw pile where it stops for each field turn, or to unload, which is often simply too big for the tractor baling to get through/over. Is there a setting somewhere which would help with this?

**Comment by Tensuko** [comment, marked as answer]
No setting for this, this. I think we have some Enhancement request for this so might be improoved in the feature.
